README.md

```diff
@@ -103,6 +103,12 @@ var responses = ["Purr", "Meow"];
 when(cat.sound()).thenAnswer((_) => responses.removeAt(0));
 expect(cat.sound(), "Purr");
 expect(cat.sound(), "Meow");
+
+// We can stub a method with multiple calls that happened in a particular order
```

Comment:
Nit: comment sentences should end with a dot.

Reply:
Thanks! I fixed the comment and the format now